opsmngr/checkpoints.go

```diff
@@ -35,7 +35,11 @@ func (s *CheckpointsServiceOp) List(ctx context.Context, groupID, clusterName st
 		return nil, nil, atlas.NewArgError("clusterName", "must be set")
 	}
 
-	path := fmt.Sprintf(checkpoints, groupID, clusterName)
+	basePath := fmt.Sprintf(checkpoints, groupID, clusterName)
+	path, err := setListOptions(basePath, listOptions)
```

Comment:
list of options should not be passed as request body but should be inserted in the path

Reply:
is this a problem in atlas as well?

Reply:
yes, it is. I am going to open a PR on atlas

Reply:
add it to this PR https://github.com/mongodb/go-client-mongodb-atlas/pull/69 so we only wait for that one